pkg/cmd/admin/node/listpods.go

```diff
@@ -107,7 +114,23 @@ func (l *ListPodsOptions) handleRESTOutput(nodes []*kapi.Node, printer kprinters
 			continue
 		}
 
-		unifiedPodList.Items = append(unifiedPodList.Items, pods.Items...)
+		unifiedPodList.ListMeta.SelfLink = pods.ListMeta.SelfLink
+
+		for _, pod := range pods.Items {
+			typedPod := &kapiv1.Pod{}
+			b, err := json.Marshal(pod)
+			if err != nil {
+				errList = append(errList, err)
+				continue
+			}
+			err = json.Unmarshal(b, typedPod)
```

Comment:
@fabianofranz not sure if we have a more streamlined way of converting from internal to external type?